app/views/contacts/new.html.haml

```diff
@@ -10,6 +15,6 @@
   .hidden
     = f.hidden_field :restroom_id, :required => false
     = f.hidden_field :restroom_name, :required => false
-    = f.hidden_field :nickname, :hint => 'Leave this field blank!'
+    = f.hidden_field :nickname, :hint => t('.leave-this-field-blank')
   .form-actions
-    = f.button :submit, 'Send message', :class=> "linkbutton"
```

Comment:
I don't see a translation here?

Reply:
It was a weird experience trying to translate this, as buttons in `simple_form` are apparently from some other dimension compared to all other parts of forms, and try as I might, I couldn't figure out a way to translate this with the usual `t('key.name.here')` format, inline in `app/views/contacts/new.html.haml`.

The translation ended up here: [simple_form.en.yml, Line 27](https://github.com/RefugeRestrooms/refugerestrooms/pull/421/files#diff-596374a12ef429714137455430e6b4c4R27)

This seemed like the logical place to put it.

I do notice that I accidentally changed the string from "_Send_ message" -> "_Submit_ message" so I can change that back if you like.